src/platforms/components/GatherPreview.tsx

```diff
@@ -0,0 +1,19 @@
+import platforms from "platforms/platforms"
+import { useWatch } from "react-hook-form"
+import PlatformPreview from "./PlatformPreview"
+
+const GatherPreview = () => {
+  const platformGuildData = useWatch({
+    name: "rolePlatforms.0.guildPlatform.platformGuildData",
+  })
+
+  return (
+    <PlatformPreview
```

Comment:
AvailabilitySetup is not shown for Gather, while it should be. We have to add it elsewhere, just commenting here